IyiKuyoro/Pals
# #161530722 setup npm project

#### What does this PR do?
This pull request creates the node package for the project

#### Tasks to be completed
> - Add the package.json file

#### How can this be manually tested
> - Clone the repo
> - Checkout into the `ch-setup-node-project-#161530722` branch
> - Check to see that `package.json` file was added.

#### Relevant PT stories
#161530722
